src/jdk.jpackage/windows/native/msiwrapper/MsiWrapper.cpp

```diff
@@ -50,8 +50,9 @@ void launchApp() {
                 (const char*)productCodeUtf8.data(), productCodeUtf8.size()));
 
         // Uninstall product.
-        msi::uninstall().setProductCode(productCode)();
-        exitCode = 0;
+        msi::SuppressUI suppressUI;
```

Comment:
Do you know why this needed and what it do?

Reply:
It is required to run uninstallation silently, without showing UI

Reply:
It is required to run uninstallation silently, without showing UI